include/ocpp/v201/messages/PublishFirmwareStatusNotification.hpp

```diff
@@ -19,8 +19,9 @@ namespace v201 {
 struct PublishFirmwareStatusNotificationRequest : public ocpp::Message {
     PublishFirmwareStatusEnum status;
     std::optional<CustomData> customData;
-    std::optional<std::vector<CiString<512>>> location;
+    std::optional<std::vector<CiString<2000>>> location;
```

Comment:
Adding a `FieldLength` variable for `PublishFirmwareStatusNotificationRequest.location` should make this backwards-compatible with 2.0.1